tests/system/action/base.py

```diff
@@ -11,23 +14,124 @@ class BaseActionTestCase(BaseSystemTestCase):
     def get_application(self) -> WSGIApplication:
         return create_action_test_application()
 
-    def request(self, action: str, data: Dict[str, Any]) -> Response:
-        return self.request_multi(action, [data])
+    def request(
+        self, action: str, data: Dict[str, Any], anonymous: bool = False
+    ) -> Response:
+        return self.request_multi(action, [data], anonymous=anonymous)
 
-    def request_multi(self, action: str, data: List[Dict[str, Any]]) -> Response:
+    def request_multi(
+        self, action: str, data: List[Dict[str, Any]], anonymous: bool = False
+    ) -> Response:
         response = self.request_json(
             [
                 {
                     "action": action,
                     "data": data,
                 }
-            ]
+            ],
+            anonymous=anonymous,
         )
         if response.status_code == 200:
             results = response.json.get("results", [])
             assert len(results) == 1
             assert len(results[0]) == len(data)
         return response
 
-    def request_json(self, payload: Payload) -> Response:
-        return self.client.post("/", json=payload)
+    def request_json(self, payload: Payload, anonymous: bool = False) -> Response:
+        client = self.client if not anonymous else self.anon_client
+        return client.post("/", json=payload)
+
+    def create_meeting(self) -> None:
+        self.set_models({"committee/1": {}})
+        response = self.request(
+            "meeting.create",
+            {
+                "committee_id": 1,
+                "name": "test",
+                "welcome_title": "title",
+            },
+        )
+        self.assert_status_code(response, 200)
+
+    def set_anonymous(self, enable: bool = True, meeting_id: int = 1) -> None:
```

Comment:
Maybe two functions: `enable_anonymous` and `disable_anonymous`.

Reply:
Mmh, I think it's fine this way